src/styles/home.scss

```diff
@@ -2,15 +2,13 @@
   text-align: center;
   width: 100%;
   color: $mainTextColor;
+  background-image: linear-gradient(180deg, #315A42, #5A5731);
```

Comment:
Looks like you have these values in your color variables. These could be cleaned up to use the variables instead